proposals/NNNN-negative-availability.md

```diff
@@ -0,0 +1,174 @@
+
+With the exception of this very specific case, an empty statement is a sure sign that there's something wrong in the developer's code, and it is likely that every unavailability check like this had to include a comment to indicate that it was done on purpose due to the compiler's limitations. In some cases, it might even be necessary to add an exclusion rule to the project's linters as most would assume that this is a mistake and incorrectly suggest that it can be fixed by negating the statement.
+
+This workaround has a clear negative impact on the readability of the unavailability check, as no one would expect an empty `if` statement to not be a coding mistake. Most developers will attempt to hide the problem by putting the positive portion in a single line:
+
+```swift
+// if NOT in iOS 13, load the window.
+// Post iOS 13 the window is loaded later in the lifecycle, in the SceneDelegate.
+if #available(iOS 13, *) {} else {
+  loadMainWindow()
+}
+```
+
+Unfortunately, this makes it easy to mistake it for a regular availability check. If the comment isn't clear that the statement is wrong on purpose, problems involving this check could easily land unnoticed in code-reviews. A less noisy way would be to put it under a `guard`:
+
+```swift
+guard #available(iOS 13, *) else {
+  loadMainWindow()
+  return
+}
+// no-op
+```
+
+However, this goes against the code style recommendations involving the usage of `guard`. The guarded part should be the happy path, which is not the case when doing unavailability checks. As shown, it's currently impossible to write an unavailability check that properly fits a developer's engineering expectations and Swift's general style guide. 
+
+Currently, any iOS application that supports UIScenes will face this issue and have to write this workaround. To describe it in a generic way, this issue will be encountered when dealing with any API addition or change that requires more than one availability condition to be implemented.
+
+## Usage of deprecated APIs
+
+A negative availability condition might also be necessary in cases where an API is marked as unavailable. While newer iOS versions will have migrated to new APIs, it's often the case that old versions must still respect the old requirements.
+
+An example is the deprecation of the [advertisingTrackingEnabled](https://developer.apple.com/documentation/adsupport/asidentifiermanager/1614148-advertisingtrackingenabled?language=objc) property. Apps supporting iOS 14 must now use the new App Tracking Transparency framework for user tracking purposes, but the property still works when used in old versions.
+
+```swift
+// If NOT on iOS 14, track this action.
+// Post iOS 14, the new privacy frameworks prevent this.
+if #available(iOS 14.0) {
+
+} else if identifierManager.advertisingTrackingEnabled {
+  // Old iOS 13 tracking logic
+}
```

Comment:
It could be, though the functionality is different enough that it would be better off placed somewhere else. What I'm seeing right now is that people are just disabling it entirely in iOS 14 with a negative check while they figure out how to implement it, so that could be the description of the section instead.